torch/_refs/__init__.py

```diff
@@ -5950,7 +5950,7 @@ def normal(
             lambda: "normal expects that either mean or std is a tensor, or shape is defined",
         )
         torch._check(
-            layout is None and pin_memory is None,
+            pin_memory is None,
             lambda: "Cannot pass layout, or pin_memory without shape",
```

Comment:
I think this genuinely should be checking `layout is None` based on the error message. Also preexisting but the keyword argument should be `size` not `shape`